pypesto/sampling/pymc3.py

```diff
@@ -104,19 +103,21 @@ def get_samples(self) -> McmcPtResult:
             self.data.posterior.to_array()).transpose((1, 2, 0))
 
         # TODO this is only the negative objective values
-        trace_fval = np.asarray(self.data.log_likelihood.to_array())
+        trace_neglogpost = np.asarray(self.data.log_likelihood.to_array())
         # remove trailing dimensions
-        trace_fval = np.reshape(trace_fval, trace_fval.shape[1:-1])
+        trace_neglogpost = np.reshape(trace_neglogpost,
+                                      trace_neglogpost.shape[1:-1])
         # flip sign
-        trace_fval = - trace_fval
+        trace_neglogpost = - trace_neglogpost
 
-        if trace_x.shape[0] != trace_fval.shape[0] \
-                or trace_x.shape[1] != trace_fval.shape[1] \
+        if trace_x.shape[0] != trace_neglogpost.shape[0] \
+                or trace_x.shape[1] != trace_neglogpost.shape[1] \
                 or trace_x.shape[2] != len(self.problem.x_names):
             raise ValueError("Trace dimensions are inconsistent")
 
         return McmcPtResult(
             trace_x=np.array(trace_x),
-            trace_fval=np.array(trace_fval),
+            trace_neglogpost=np.array(trace_neglogpost),
+            trace_neglogprior=np.full(trace_neglogpost.shape, np.nan),
```

Comment:
yes, can't do anything about this atm. maybe something for @lcontento to be aware of.